uber/templates/registration/menu.html

```diff
@@ -1 +1,5 @@
+{% if c.HAS_WATCHLIST_ACCESS %}
```

Comment:
Bleh, the ``nav_menu`` plugin should probably be replaced by something less ugly.  For now this change looks good, given the limitations of ``nav_menu``.